src/style.scss

```diff
@@ -123,6 +123,21 @@
         }
       }
     }
+    // CTA ondark
+    &.hyperlink--ondark {
+      background-color: var(--auro-color-ui-default-on-dark);
+      border: 1px solid var(--auro-color-ui-default-on-dark);
+      color: var(--auro-color-text-primary-on-light);
+
+      // update to media query when ods-hyperlink support is removed
+      :host(:not(.is-touching)) & {
+        &:hover {
+          background-color: var(--auro-color-ui-hover-on-dark);
+          border: 1px solid var(--auro-color-ui-hover-on-dark);
+          color: var(--auro-color-text-primary-on-light);
+        }
+      }
+    }
```

Comment:
I think you are missing a `:visited` style

<img width="1353" alt="Screen Shot 2021-02-22 at 9 22 40 PM" src="https://user-images.githubusercontent.com/181089/108805149-3281c480-7554-11eb-810e-657f36973cdf.png">